common/unified/multigrid/pgm_kernels.cpp

```diff
@@ -311,6 +311,22 @@ GKO_INSTANTIATE_FOR_EACH_NON_COMPLEX_VALUE_AND_INDEX_TYPE(
     GKO_DECLARE_PGM_ASSIGN_TO_EXIST_AGG);
 
 
+template <typename IndexType>
+void gather_index(std::shared_ptr<const DefaultExecutor> exec,
```

Comment:
nit:
```suggestion
void gather_indices(std::shared_ptr<const DefaultExecutor> exec,
```

Reply:
we also use `gather_index` in dense, so I keep the same name